packages/terra-slide-panel-manager/src/SlidePanelManager.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import AppDelegate from 'terra-app-delegate';
+import DisclosureManager, { availableDisclosureSizes } from 'terra-disclosure-manager';
+import SlideGroup from 'terra-slide-group';
+import SlidePanel from 'terra-slide-panel';
+
+const propTypes = {
+  /**
+   * An AppDelegate instance that will be used in fallback scenarios by the SlidePanelManager. If not provided, the SlidePanelManager
+   * will handle any and all disclosure requests it receives.
+   */
+  app: AppDelegate.propType,
+  /**
+   * The components to display in the body of the SlidePanel. These components will be provided with an AppDelegate prop (as `app`) that
+   * will allow them to disclose content.
+   */
+  children: PropTypes.node,
+  /**
+   * The desired panel behavior. Either 'squish' or 'overlay'.
```

Comment:
Have you heard anything more about 'squish'? Will be still support this?

Reply:
Yeah I'll make overlay default. Squish is still supported as I haven't heard otherwise, though things might change.